### ISSUE TITLE
Does BEIR support tuning config k and b of BM25 ?

### ISSUE THREAD

**Issue by Yuan0320** [beir-cellar/beir#103]
Hi, thanks for the amazing work. I would like to know if BEIR supports tuning config k and b of BM25?
In the paper, anserini BM25 was implemented with default Lucene parameters (k=0.9 and b=0.4). I want to run [evaluate_bm25.py](https://github.com/beir-cellar/beir/blob/main/examples/retrieval/evaluation/lexical/evaluate_bm25.py) or  [evaluate_anserini_bm25.py](https://github.com/beir-cellar/beir/blob/main/examples/retrieval/evaluation/lexical/evaluate_anserini_bm25.py) of beir with different k and b. But I can't find where to tune them.

**Comment by lintool** [comment]
You can find Anserini regressions for BEIR here: https://github.com/castorini/anserini#regressions-for-beir-v100
You can reuse the tuning script implemented here: https://github.com/castorini/anserini/blob/master/docs/experiments-msmarco-passage.md#bm25-tuning

**Comment by Jia-py** [comment]
> You can find Anserini regressions for BEIR here: https://github.com/castorini/anserini#regressions-for-beir-v100 You can reuse the tuning script implemented here: https://github.com/castorini/anserini/blob/master/docs/experiments-msmarco-passage.md#bm25-tuning

Does BEIR support originally setting the k and b for BM25, or should I import the BM25 from Anserini to set these parameters? I find the scripts in the two links are hard to follow. Are there any other relevant tutorials or documentation?

Thanks
